metrics/web/test_driver/arguments/model/driver_test_arguments.dart

```diff
@@ -16,6 +17,7 @@ class DriverTestArguments {
   ///
   /// [workingDir] is the directory to run all the commands.
   /// [logsDir] is the directory to store the logs from commands.
+  /// [chromedriverVersion] is the version of the chromedriver.
```

Comment:
So, as we've touched this file, should we refactor the documentation here to be over each field instead of the constructor? 🙂